security/tokens_and_compartments.rst

```diff
@@ -0,0 +1,199 @@
+        "FilterArgument": "general-practitioner.identifier=#patient#" //... for the Patient that has an identifier matching the value of that 'patient' launch scope
+      },
+      ...
+   ]
+
+In this example the claim is still called 'patient', although it contains an Identifier of a General Practitioner. 
+This is because the CompartmentDefinition is selected by matching its code to the name of the claim, regardless of the value the claim contains.
+
+.. note::
+   Any request is scoped to the patient compartment and requests are rejected if the patient claim is not provided in the access token.
+
+.. _feature_accesscontrol_decisions:
+
+Access Control Decisions for Patient-level scopes
+-------------------------------------------------
+
+In this paragraph we will explain how access control decisions are made for the various FHIR interactions. For the examples assume a Patient Compartment with identifier=123 as filter.
+
+#. Search
+
+   a. Direct search on compartment type
+
+      :Request: ``GET [base]/Patient?name=fred``
+      :Type-Access: User must have read access to Patient, otherwise HTTP Status Code 403 is returned. 
+      :Compartment: If a Patient Compartment is active, the Filter from it will be added to the search, e.g. ``GET [base]/Patient?name=fred&identifier=123``
+
+   #. Search on type related to compartment
+
+      :Request: ``GET [base]/Observation?code=x89``
+      :Type-Access: User must have read access to Observation, otherwise HTTP Status Code 403 is returned. 
+      :Compartment: If a Patient Compartment is active, the links from Observation to Patient will be added to the search. In pseudo code: ``GET [base]/Observation?code=x89& (subject:Patient.identifier=123 OR performer:Patient.identifier=123)``
+
+   #. Search on type not related to compartment
+
+      :Request: ``GET [base]/Organization``
+      :Type-Access: User must have read access to Organization, otherwise HTTP Status Code 403 is returned. 
+      :Compartment: No compartment is applicable to Organization, so no further filters are applied.
+
+   #. Search with include outside the compartment
+
+      :Request: ``GET [base]/Patient?_include=Patient:organization``
+      :Type-Access: User must have read access to Patient, otherwise HTTP Status Code 403 is returned. If the user has read access to Organization, the _include is evaluated. Otherwise it is ignored.
+      :Compartment: Is applied as in case 1.a.
+
+   #. Search with chaining
+
+      :Request: ``GET [base]/Patient?general-practitioner.identifier=123``
+      :Type-Access: User must have read access to Patient, otherwise HTTP Status Code 403 is returned. If the user has read access to Practitioner, the search argument is evaluated. Otherwise it is ignored as if the argument was not supported. If the chain has more than one link, read access is evaluated for every link in the chain. 
```

Comment:
In this case 'ignored' will result in an error, so the request will not be authorized. We may need to make this explicit since in a normal search 'ignored' would result in all Patients matching...